.github/workflows/lint.yaml

```diff
@@ -0,0 +1,22 @@
+name: lint
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
```

Comment:
Have you considered adding concurrency setting like [so](https://github.com/youtube/cobalt_sandbox/blob/main/.github/workflows/main.yaml#L34) to have in-progress presubmit workflow cancelled if new commit is pushed to PR?

Reply:
Done